app/src/main/res/values/anim.xml

```diff
@@ -19,5 +19,5 @@
 <resources>
 
     <!-- Duration (in milliseconds) for slide transition animation -->
-    <integer name="slide">700</integer>
+    <integer name="slide">400</integer>
```

Comment:
I don't want to hinder your thoughts, but I also recommend the following:
<img width="420" alt="screen shot 2019-02-12 at 22 28 35" src="https://user-images.githubusercontent.com/7869833/52669187-bdfffe00-2f15-11e9-9e8a-37b5fb610b5c.png">

Reply:
Oh, it seems like config_longAnimTime also equals to 400. In my opinion, having our own set value might enhance readability and flexibility if the duration requirement changes in the future. But I can update the PR, if this is required by all the reviewers.

Reply:
Hm... I wonder if using `@android:integer/config_mediumAnimTime` might be a good choice then?

Reply:
Thanks for point that out, @XinyueZ.  Note that these config values may change with OS releases.  So depending on your needs it may make sense to match the framework's animation values.

For our purposes let's keep it at 400.